--- epel.repo
+++ epel.repo
@@ -1,5 +1,6 @@
 [epel]
 name=Extra Packages for Enterprise Linux 7 - $basearch
+exclude=tor
 #baseurl=http://download.fedoraproject.org/pub/epel/7/$basearch
 mirrorlist=https://mirrors.fedoraproject.org/metalink?repo=epel-7&arch=$basearch
 failovermethod=priority
@@ -9,6 +10,7 @@
 
 [epel-debuginfo]
 name=Extra Packages for Enterprise Linux 7 - $basearch - Debug
+exclude=tor
 #baseurl=http://download.fedoraproject.org/pub/epel/7/$basearch/debug
 mirrorlist=https://mirrors.fedoraproject.org/metalink?repo=epel-debug-7&arch=$basearch
 failovermethod=priority
@@ -18,6 +20,7 @@
 
 [epel-source]
 name=Extra Packages for Enterprise Linux 7 - $basearch - Source
+exclude=tor
 #baseurl=http://download.fedoraproject.org/pub/epel/7/SRPMS
 mirrorlist=https://mirrors.fedoraproject.org/metalink?repo=epel-source-7&arch=$basearch
 failovermethod=priority
