--- config.def.h	2016-05-25 22:09:31.975915787 -0600
+++ config.def.h	2016-05-25 22:14:39.975464616 -0600
@@ -5,7 +5,8 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char font[] = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+//static char font[] = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char font[] = "mplus Nerd Font:style=medium:size=13";
 static int borderpx = 2;
 
 /*
