*** SRC/thttpd-2.27/Makefile.in	2005-05-20 13:10:24.000000000 -0600
*************** installthis:
*** 113,119 ****
  	$(INSTALL) -m 555 -o bin -g bin thttpd $(DESTDIR)$(BINDIR)
  
  install-man:
! 	-mkdir -p $(DESTDIR)$(MANDIR)/man8
  	$(INSTALL) -m 444 -o bin -g bin thttpd.8 $(DESTDIR)$(MANDIR)/man8
  
  installsubdirs:
--- 113,120 ----
  	$(INSTALL) -m 555 -o bin -g bin thttpd $(DESTDIR)$(BINDIR)
  
  install-man:
! 	mkdir -p $(DESTDIR)$(MANDIR)/man1
! 	mkdir -p $(DESTDIR)$(MANDIR)/man8
  	$(INSTALL) -m 444 -o bin -g bin thttpd.8 $(DESTDIR)$(MANDIR)/man8
  
  installsubdirs:
